python/cudaq/__init__.py

```diff
@@ -54,6 +54,15 @@
     from .visualization.bloch_visualize import add_to_bloch_sphere
     from .visualization.bloch_visualize import show_bloch_sphere as show
 
+
+def __getattr__(name):
+    if hasattr(cudaq_runtime, name):
+        return getattr(cudaq_runtime, name)
+    raise AttributeError(f"'{name}' is not supported for CUDA-Q in Python")
```

Comment:
The real issue is that there are functions that can only be called within cuda-q kernel code and not from host code. its not really about the function not being supported in Python.